meta-genivi-dev/meta-genivi-dev/recipes-dev-platform/images/genivi-dev-platform.bb

```diff
@@ -38,6 +38,7 @@ IMAGE_INSTALL_append = " \
     packagegroup-gdp-rvi \
     packagegroup-gdp-vsi \
     packagegroup-gdp-sdl \
+    packagegroup-gdp-vsi \
```

Comment:
I was wondering about this commit since it seems unrelated to the function.  The commit seems already merged on master.   Somehow you doubled it - the same commit information is found twice in the commit history of this (PR) branch.